client/util/util.go

```diff
@@ -63,6 +64,17 @@ func LoadSignerBinaryPath(configFilePath string) (path string, err error) {
 	if signerBinaryPath == "" {
 		return "", ErrConfigUnavailable
 	}
+
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return "", err
+	}
+	// Checking for the "~" or "$HOME" variables to represent home directory
+	if signerBinaryPath[0] == '~' {
+		signerBinaryPath = strings.ReplaceAll(signerBinaryPath, "~", homeDir)
```

Comment:
I would replace "homeDir" with a direct function call to guessHomeDir() here and below, so you don't need lines 68-71. The guessHomeDir is preferred way to determine home directory for legacy reasons - use same function to be consistent.